### DISCUSSION TITLE
[Realtime] Subcribe to multiple ID's

### DISCUSSION THREAD

**Discussion by moisesBolano** [pocketbase/pocketbase#3350] [Q&A] [answered]
Hello Guys.

Im trying to subscribe @ multiple ID's of collection.

My code:

`pb.collection('myCollection').subscribe(['ID1', 'ID2'], (e) => {})`

**Comment by ganigeorgiev** [comment]
At the moment you can call `subscribe(id, ...)` in a loop for each id OR subscribe to `*` and filter the specific records from inside the callback (note that when subscribing to `*` the client will receive events **only** for the modified records that satisfy the List/Search API rule of the collection).

**Reply by moisesBolano** [reply, marked as answer]
Working code:  

`const IDS = ['ID1', 'ID2'];

  const handleQr = (e) => {
    //
  }

  IDS.forEach(qr => {
    pb.collection('myCollection').subscribe(qr, (e) => handleEvent(e));  
  });
`

Thanks @ganigeorgiev
